### PR TITLE
Portfolio week 8 

### PR DESCRIPTION
**Pull request by Yennie-Leek**
I have created my first portfolio. 
Its mostly just a structure for now and I will add more specific info further on to update it.
This week has been a struggle, see more in the ReadMe file.

### CODE REVIEW

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
You didn't provide a link to the hosted version of your site. Please do it ASAP to make me possible to test it.

**Comment by Yennie-Leek** [comment]
Thank you for the info! Link to live site is now updated in ReadMe

File: index.HTML [hunk 1]
```diff
@@ -0,0 +1,333 @@
...
+        <div tabindex="0" class="summary">
+          <img
+            class="textwrap"
+            src="./Assets/icons8-dreamcatcher-50.png"
+            alt="Dream Catcher"
+          />
+          <p>
+            Phasellus ullamcorper varius arcu vitae fringilla. Integer consequat
+            lacinia ligula in dignissim. In dapibus, magna scelerisque vulputate
+            porttitor, elit leo eleifend justo, ac lobortis odio ligula at eros.
```
Comment (KarolCislowski) [inline index.HTML]: This section is about you, so write a few words about yourself.

File: index.HTML [hunk 2]
```diff
@@ -0,0 +1,333 @@
...
+            <div>
+              <span><mark>HTML 5</mark></span>
+              <span><mark>CSS3</mark></span>
+              <span><mark>JavaScript ES6</mark></span>
+            </div>
+          </a>
+        </section>
+
+        <section class="top-project">
+          <a href="a">
```
Comment (KarolCislowski) [inline index.HTML]: Please provide links to the hosted versions of the projects.

File: index.HTML [hunk 3]
```diff
@@ -0,0 +1,333 @@
...
+            <img src="./Assets/Chat-Bot.png" alt="Closet Chat Bot" />
+            <h5>Chat Bot Project</h5>
+            <p>
+              Praesent ultrices vestibulum erat in lacinia. Etiam quis finibus
+              nibh. Aenean quis accumsan diam.
```
Comment (KarolCislowski) [inline index.HTML]: Please provide descriptions of your projects.

File: index.HTML [hunk 4]
```diff
@@ -0,0 +1,333 @@
...
+              <span><mark>CSS3</mark></span>
+              <span><mark>JavaScript ES6</mark></span>
+            </div>
+          </a>
+        </section>
+
+        <section class="top-project">
+          <a href="a">
+            <img src="./Assets/Chat-Bot.png" alt="Closet Chat Bot" />
+            <h5>Chat Bot Project</h5>
```
Comment (KarolCislowski) [inline index.HTML]: The name of the project needs to be located on top of your picture. 🎨

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Only one thing left to do - fill this nice looking page with your own content and share to the word 😸

**Review by KarolCislowski — APPROVED** [review]
Great Project 😸

**Pull request closed without merging**